module #4. Authentication & Spring Security/oauth2-server/src/main/java/com/epam/esm/GiftCertificateOAuth2ServerApplication.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.esm;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class GiftCertificateOAuth2ServerApplication {
```

Comment:
this auth service can be used for uaa at different services, not only for gift certificates